ophidian/floorplan/Floorplan.cpp

```diff
@@ -0,0 +1,67 @@
+#include "Floorplan.h"
+
+namespace ophidian {
+
+namespace floorplan {
+
+Floorplan::Floorplan()
+    : chipOrigin_(0.0, 0.0), chipUpperRightCorner_(0.0, 0.0),
+      origins_(rows_), numberOfSites_(rows_), siteTypeOfRow_(rows_),
+      names_(sites_), dimensions_(sites_)
+{
+
+}
+
+Floorplan::~Floorplan()
+{
+
+}
+
+void Floorplan::chipOrigin(util::Location loc)
+{
+    chipOrigin_ = loc;
+}
+
+void Floorplan::chipUpperRightCorner(util::Location loc)
+{
+    chipUpperRightCorner_ = loc;
+}
+
+Site Floorplan::add(Site, std::string name, util::Location loc)
```

Comment:
I think all non-primitive types should be passed as reference. If they will not be modified, then use  const, of course!